### PR TITLE
 databinding home screen

### PR DESCRIPTION
**Pull request by meoconlontonbebong**
LInk Redmi : https://edu-redmine.framgia.vn/issues/6802
 Binding Homes Screen

### CODE REVIEW

**Comment by meoconlontonbebong** [comment]
ready

File: app/src/main/java/com/example/moviedb_37/screen/home/CategoryAdapter.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    public void onBindViewHolder(@NonNull ViewHolder viewHolder, int i) {
+        viewHolder.bindData(mMovies.get(i));
+    }
+
+    @Override
+    public int getItemCount() {
+        return mMovies != null ? mMovies.size() : 0;
+    }
+
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/moviedb_37/screen/home/CategoryAdapter.java]: remove bl

File: app/src/main/java/com/example/moviedb_37/screen/home/CategoryAdapter.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    public static class ViewHolder extends RecyclerView.ViewHolder {
+        private ItemMovieBinding mBinding;
+        private ItemMovieListViewModel mItemMovieListViewModel;
+
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/moviedb_37/screen/home/CategoryAdapter.java]: remove bl

File: app/src/main/java/com/example/moviedb_37/screen/home/GenresAdapter.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+import com.example.moviedb_37.R;
+import com.example.moviedb_37.data.model.Genre;
+import com.example.moviedb_37.databinding.ItemGenreBinding;
+
+import java.util.List;
+
+public class GenresAdapter extends RecyclerView.Adapter<GenresAdapter.ViewHolder> {
+
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/moviedb_37/screen/home/GenresAdapter.java]: remove bl

File: app/src/main/res/layout/item_genre.xml [hunk 4]
```diff
@@ -61,3 +67,4 @@
             app:layout_constraintTop_toBottomOf="@id/text_movie_name" />
     </android.support.constraint.ConstraintLayout>
 </android.support.v7.widget.CardView>
+</layout>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/item_genre.xml]: need an endline

File: app/src/main/java/com/example/moviedb_37/screen/home/HomeViewModel.java [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+import java.util.List;
+
+public class HomeViewModel extends BaseObservable {
+
+    public ObservableBoolean mIsLoadMorePopular = new ObservableBoolean();
+    public ObservableBoolean mIsLoadMoreNowPlaying = new ObservableBoolean();
+    public ObservableBoolean mIsLoadMoreTopRate = new ObservableBoolean();
+    public ObservableBoolean mIsLoadMoreUpComing = new ObservableBoolean();
+
+    private CategoryAdapter mPopularAdapter, mNowPlayingAdapter, mTopRateAdapter, mUpComingAdapter;
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/moviedb_37/screen/home/HomeViewModel.java]: mỗi biến 1 dòng

**Comment by meoconlontonbebong** [comment]
ready

**Pull request merged**